implementations/go-jsonschema/bowtie_jsonschema.go

```diff
@@ -45,30 +46,15 @@ func main() {
 			if version != 1 {
 				panic("Not version 1!")
 			}
-
-			buildInfo, ok := debug.ReadBuildInfo()
-			if !ok {
-				panic("Failed to read build info")
-			}
-
-			var jsonschemaVersion = ""
-			for _, dep := range buildInfo.Deps {
-				if strings.Contains(dep.Path, "github.com/santhosh-tekuri/jsonschema") {
-					jsonschemaVersion = dep.Version
-					break
-				}
-			}
-
 			data := map[string]interface{}{
 				"ready":   true,
-				"version": 1,
+				"version": version,
```

Comment:
I want this explicitly hardcoded, the point is that a harness shouldn't just blindly copy the version that comes in, it has to be sure it's indeed speaking the right protocol (because e.g. someone will come in here and change support to v2 if the protocol changes, and then they update this from 1 to 2). But if bowtie starts speaking v2 and no changes happen here, the harness shouldn't suddenly claim it's speaking v2, it probably isn't.

Reply:
we already panic if version!=1. also when updating to version 2 in future we don't worry about changing version in `if condition` and also in `json response`

Reply:
Perhaps a middle ground then is to just have a global `VERSION = 1` which is defined in one place and then use that both for `panic`'ing and for the JSON response? That will save updating many places. This isn't major, but again, the issue I have is partially that it's not correct that the version here is the one from the request, which is what reusing the variable would indicate semantically -- the version here is the one that the harness is written for, which yes if the code is all correct and we paniced will always match the request, but it's asking for some subtle issue maybe not in this harness but in some future one that forgets to panic.